x-pack/packages/kbn-entities-schema/src/schema/common.ts

```diff
@@ -88,6 +88,11 @@ export const metadataSchema = z
     destination: z.optional(z.string()),
     limit: z.optional(z.number().default(1000)),
   })
+  .transform((metadata) => ({
+    ...metadata,
+    desitination: metadata.destination ?? metadata.source,
```

Comment:
```suggestion
    destination: metadata.destination ?? metadata.source,
```

Reply:
neat, can we get rid of the destination fallbacks in [processors](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/entity_manager/server/lib/entities/ingest_pipeline/generate_latest_processors.ts#L29) and [aggregations](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/entity_manager/server/lib/entities/transform/generate_metadata_aggregations.ts#L18) logic since it's now guaranteed to be set ?

Reply:
also should we use `||` instead of `??` in case destination is set to an empty string ? same for limit in case it's set to 0

Reply:
Done